sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+            education_programs.noc_code as "National Occupation Classification",
+            education_programs.sabc_code as "SABC Program Code",
+            education_programs.institution_program_code as "Institution Program Code",
+            (
+                case
+                    when education_programs.program_intensity = ''Full Time and Part Time'' then ''Y''
+                    else ''N''
+                end
+            ) as "Part-Time eligible (Yes/No)",
+            ''--TBD--'' as "Program Delivery ",
+            education_programs.completion_years as "Program Length",
+            ''Credit or Hours Based'' as "Credit or Hours Based",
+            ''--TBD--'' as "Program Regulatory Body",
+            ''--TBD--'' as "Entrance Requirements",
+            ''--TBD--'' as "Program ESL Content Percentage",
+            ''--TBD--'' as "Program offered jointly/ in partnership (Yes/No)",
+            ''--TBD--'' as "Program Work Integrated Learning (WIL) Component (Yes/No)",
+            ''--TBD--'' as "WIL regulatory approval (Yes/No � as applicable)",
+            ''--TBD--'' as "Program Field Trip / Travel Component (Yes/No)",
+            ''--TBD--'' as "Program international exchange component (Yes/No)",
+            ''--TBD--'' as "Program was pending but then aproved",
+            -- Study Period Offering Info
+            education_programs_offerings.offering_name as "Study Period Offering Name",
+            education_programs_offerings.year_of_study as "Year of Study",
+            education_programs_offerings.offering_intensity as "Offering intensity",
+            ''--TBD--'' as "Offering course load percentage",
+            education_programs_offerings.offering_delivered as "Offering delivery method",
+            ''--TBD--'' as "Offering WIL component (Yes/No)",
+            to_char(
+                education_programs_offerings.study_start_date,
+                ''YYYY-MM-DD''
+            ) as "Study Start date",
+            to_char(
+                education_programs_offerings.study_end_date,
+                ''YYYY-MM-DD''
+            ) as "Study End Date",
+            ''--TBD--'' as "Study Break Start Date(s)",
+            ''--TBD--'' as "Study Break End Date(s)",
+            ''--TBD--'' as "Total Offering Days",
+            ''--TBD--'' as "Total Funded Weeks",
+            ''--TBD--'' as "Total Funded Study Period Days",
+            ''--TBD--'' as "Total Unfunded Study Period Days",
+            education_programs_offerings.actual_tuition_costs as "Tuition",
+            education_programs_offerings.program_related_costs as "Books and Supplies",
+            education_programs_offerings.mandatory_fees as "Mandatory Fees",
+            education_programs_offerings.exceptional_expenses as "Exceptional Costs",
+            ''--TBD--'' as "Offering availability",
+            ''--TBD--'' as "Offering pending flag",
+            -- Scholastic Standing Info
+            ''--TBD--'' as "Early completion date",
```

Comment:
As discussed adding the fields for the next story,

![image](https://github.com/bcgov/SIMS/assets/77353155/e9c1dd42-d890-4d01-9389-27f2284334ae)